examples/amp-story/interactive_results.html

```diff
@@ -0,0 +1,155 @@
+                prompt-text="What would you do for holidays?"
+                option-1-text="Easter island" option-1-category="Bunny" option-1-correct
+                option-2-text="Drive in a car with your tongue out" option-2-category="Dog" option-2-correct
+                option-3-text="Get far away from cats" option-3-category="Mouse" option-3-correct
+                option-4-text="Stay in bed sleeping" option-4-category="Cat" option-4-correct>
+            </amp-story-interactive-quiz>
+          </amp-story-grid-layer>
+        </amp-story-page>
+        <amp-story-page id="quiz-4">
+          <amp-story-grid-layer template="fill">
+            <amp-img
+                src="img/green-stuff.jpg" layout="responsive" height="1600" width="900">
+            </amp-img>
+          </amp-story-grid-layer>
+          <amp-story-grid-layer template="center">
+            <amp-story-interactive-quiz
+                prompt-text="What's your favorite place in the house?"
+                option-1-text="Inside my spacious cage" option-1-category="Mouse" option-1-correct
+                option-2-text="On the backyard" option-2-category="Bunny" option-2-correct
+                option-3-text="Where my owner is" option-3-category="Dog" option-3-correct
+                option-4-text="On top of the fridge" option-4-category="Cat" option-4-correct>
+            </amp-story-interactive-quiz>
+          </amp-story-grid-layer>
+        </amp-story-page>
+        <amp-story-page id="quiz-results">
+          <amp-story-grid-layer template="fill">
+            <amp-img
+                src="https://www.positive.news/wp-content/uploads/2019/03/feat-1800x0-c-center.jpg"
+                layout="responsive"
+                height="1600"
+                width="900">
+            </amp-img>
+          </amp-story-grid-layer>
+          <amp-story-grid-layer>
+            <amp-story-interactive-results-category
+								style="--interactive-accent-color: #F34E4E"
+								prompt-text="You are a"
+                option-1-category="Cat" option-1-text="Everyone loves cats, and so do you! Cats are the best companion to WFH" option-1-image="https://images.pexels.com/photos/104827/cat-pet-animal-domestic-104827.jpeg"
+                option-2-category="Dog" option-2-text="You always have energy and love being with friends. Outdoors is your favorite place" option-2-image="https://images.pexels.com/photos/1108099/pexels-photo-1108099.jpeg"
+                option-3-category="Bunny" option-3-text="You love jumping around and having fun. Your bubbly personality is contagious, and your love for carrots is unconditional" option-3-image="https://images.pexels.com/photos/326012/pexels-photo-326012.jpeg"
+                option-4-category="Mouse" option-4-text="Even though you're small, you're also full of curiosity and love going around finding new things to do" option-4-image="https://images.pexels.com/photos/51340/rat-pets-eat-51340.jpeg">
```

Comment:
This is the API being used, each category has a name (option-#-category), image and text.
PS: all images are from Pexels, which are CC0